doc/protocols.rst

```diff
@@ -386,3 +386,12 @@ References: `Protocol Specs: <https://opennetworking.org/wp-content/uploads/2014
 JSON-RPC is a remote procedure call protocol encoded in JSON.
 
 References: `Protocol Specs: <https://www.jsonrpc.org/specification>`_.
+
+
+.. _Proto 376:
+
+`NDPI_PROTOCOL_WEBDAV`
+======================
+WebDAV is a set of extensions to the HTTP protocol that allows WebDAV clients to collaboratively edit and manage files on remote Web servers.
+
+References: `RFC4918: <https://datatracker.ietf.org/doc/html/rfc4918>`_.
```

Comment:
Could you add a note (similar to HTTP2 and to your commit message) about the fact that it is unlikely to find it in cleartext?